app/javascript/bundles/Dashboard/components/SettingsComponents/DisplaySettings/Background.jsx

```diff
@@ -1,19 +1,24 @@
 import React, { useState, useEffect } from 'react'
 import style from './Background.module.scss'
 import { Radio } from 'antd';
-import { handleFormSubmit } from '../../../utils/formUtils';
+import axios from 'axios';
 
-const Background = () => {
+const Background = ({onUpdateSuccess}) => {
 	const [selectedBackground, setSelectedBackground] = React.useState(null);
 	const [authenticity_token] = useState(document.querySelector('meta[name="csrf-token"]').content);
 
-  const onChange = (e) => {
+  const onChange = async (e) => {
     const newBackground = e.target.value;
 		setSelectedBackground(newBackground);
 		const url = "/users";
-		const method = "patch";
-		const data = {"user[background_path]": newBackground};
-		handleFormSubmit(url, method, authenticity_token, data);
+		const data = {"background_path": newBackground};
+		try {
+			const response = await axios.patch(url, data, { headers: { 'X-CSRF-Token': authenticity_token } });
```

Comment:
You should be able to set this once, shouldn't need to set in every axios request
https://github.com/shakacode/react_on_rails/blob/master/docs/api/javascript-api.md